app/scripts/components/collections/list.js

```diff
@@ -50,9 +51,14 @@ class CollectionList extends React.Component {
   }
 
   generateBulkActions () {
+    const actionConfig = {
+      recover: {
+        action: applyRecoveryWorkflowToCollection
```

Comment:
I'm sure I'm missing something, but how does this action receive the collection ID?